tss-esapi/src/structures/lists/digest_values.rs

```diff
@@ -24,14 +24,40 @@ impl DigestValues {
     }
 }
 
+fn digest_to_ha(digest: (HashingAlgorithm, Digest)) -> Result<TPMT_HA> {
```

Comment:
What is the reason for this move from hash/agile?

Reply:
Yes, I forgot to mention that - had to change it because `TPMT_HA` is now defined in a different crate, and because there was a different generic implementation of the `TryFrom`, that I wasn't sure how to solve apart from switching to this

Reply:
I wonder if it would have been better to create a specific type instead of the tuple or if that would just have given rise to more problems.

Reply:
I am ok with things as they are. But it is a little bit inconsistent to have a function doing the conversion instead of a try_from.

Reply:
Done! I've created a new structure, `HashAgile` in the place of the previous `TryFrom`